eigen3/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+load(
+    "//ign_bazel:build_defs.bzl",
+    "IGNITION_ROOT",
+    "IGNITION_VISIBILITY",
+)
+
+package(
+    default_visibility = IGNITION_VISIBILITY,
+    features = [
+        "-layering_check",
```

Comment:
nit: `-layering_check` is not necessary in this case.

Reply:
Removed  in e7a37b4